tools/release/release-output/output-validations.ts

```diff
@@ -87,3 +87,12 @@ export function checkMaterialPackage(packagePath: string): string[] {
 
   return failures;
 }
+
+/**
+ * Checks whether the prebuilt CDK files are part of the release output.
+ */
+export function checkCdkPackage(packagePath: string): string[] {
+  return ['overlay', 'a11y', 'text-field']
+      .filter(name => !existsSync(join(packagePath, `${name}-prebuilt.css`)))
```

Comment:
Cool! I would have needed something like that for https://github.com/angular/material2/pull/15497.

Feels a bit weird to hardcode the entry-points. Maybe we can use a glob and look for SCSS files in a given entry-point? Also how about also checking that e.g. `_a11y.scss` is exposed at the root? (I initially missed this in #15497)

Reply:
I'm not sure I follow the note about using a Glob. As for hardcoding the entry points, I don't think it's an issue, because we have a limited set of prebuilt styles that we're exposing.

Reply:
My concern is just that whenever we add a new style file to a CDK package, we need to remember adding the entry-point name to that array.

For globs: I meant that we just look for any SCSS files in the format of  `src/cdk/{name}/_{name}.scss`. That way we could avoid the hard-coded array of entry-points that generate a prebuilt CSS file.

Reply:
That is a little brittle though. E.g. it won't handle the case where `overlay` is generated, but `a11y` or `text-field` aren't.

Reply:
Maybe I don't follow what you are saying, but my stance is that:

* If there is a `{name}-prebuilt.scss` file, then it will be exposed at the root of the CDK
* If there is a `_{name}.scss` file, then it will be exposed at the root of the CDK.

This is basically how the package-tools works: [see here](https://github.com/angular/material2/blob/master/tools/package-tools/build-release.ts#L128-L129). I don't feel too strong about it.

Reply:
My point was that the glob only verifies that some file with that pattern is generated, it won't verify that the exact files we're expecting are in the directory.

Reply:
Sorry if my comment was ambiguous. My point was that you could use the glob to _identify_ the files in the source folder and then check if the corresponding output files exist in the release output.